From 603de58f0e777c0191f78a2a01821089e62aeacb Mon Sep 17 00:00:00 2001
From: Lim Siew Hoon <siew.hoon.lim@intel.com>
Date: Mon, 22 Jul 2024 16:14:04 +0800
Subject: [PATCH 90/98] vavp9enc: delete the useless frame counter fields

They are used to calculate the PTS and DTS before, no usage now.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/6773>

Signed-off-by: Lim Siew Hoon <siew.hoon.lim@intel.com>
---
 sys/va/gstvavp9enc.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/sys/va/gstvavp9enc.c b/sys/va/gstvavp9enc.c
index c97f1cf60214..416afc52f9b8 100644
--- a/sys/va/gstvavp9enc.c
+++ b/sys/va/gstvavp9enc.c
@@ -160,8 +160,6 @@ struct _GstVaVp9EncFrame
   gint frame_num;
   guint32 flags;
   guint pyramid_level;
-  /* The total frame count we handled. */
-  guint total_frame_count;
   gboolean bidir_ref;
   gint8 ref_frame_idx[GST_VP9_REF_FRAME_MAX];
   /* The index in reference list to update */
@@ -267,7 +265,6 @@ gst_va_vp9_enc_frame_new (void)
   frame->frame_num = -1;
   frame->type = FRAME_TYPE_INVALID;
   frame->base.picture = NULL;
-  frame->total_frame_count = 0;
   frame->pyramid_level = 0;
   frame->flags = 0;
   frame->bidir_ref = FALSE;
@@ -294,7 +291,6 @@ gst_va_vp9_enc_new_frame (GstVaBaseEnc * base, GstVideoCodecFrame * frame)
   GstVaVp9EncFrame *frame_in;
 
   frame_in = gst_va_vp9_enc_frame_new ();
-  frame_in->total_frame_count = base->input_frame_count++;
   gst_va_set_enc_frame (frame, (GstVaEncFrame *) frame_in,
       gst_va_vp9_enc_frame_free);
 
@@ -2745,8 +2741,6 @@ gst_va_vp9_enc_prepare_output (GstVaBaseEnc * base,
   gst_buffer_replace (&frame->output_buffer, buf);
   gst_clear_buffer (&buf);
 
-  base->output_frame_count++;
-
   return TRUE;
 }
 
-- 
2.43.0

